mk/README.md

```diff
@@ -0,0 +1,152 @@
+
+@Variables
+
+- `mk/projectdir.mk`: It uses to be the first include and just store
+  at the `PROJECT_DIR` variable the path to the repository in the
+  file system.
+- `mk/private.mk`: Not used into this project, but it could define
+  by using conditional assignment values that we want to set for
+  our make rules; useful for the specific rules related with our
+  tech-stack, or when building container images, to customize
+  our own registry (quay.io/my-username); provide a mechanism
+  of customization to avoid repeat steps manually as `export MY_VAR=my-value`.
+- `mk/variables.mk`: It defines the default values for variables.
+  The intention is this file is kept with no changes into the
+  repository, and any change into the values should be overrided
+  by the `mk/private.mk` file or the environment variables.
+
+@Help infrastructure
+
+- `mk/meta-*.mk`: This files is just one line content each one,
+  but that allow us to print the group label into the help generated
+  as we want. It can not be written directly into the `mk/includes.mk`
+  file because of the order the files are processed when the `help`
+  rule is invoked.
+- `mk/help.mk`: Contain the rule that generate the help content.
+
+@Rules
+
+- `mk/go-rules.mk`: Provide useful rules to build/clean/test
+  a golang project. It provides generic rule to build any
+  `cmd/COMMAND` directory into the `OUTPUT` directory putting
+  there the generated binary. So if we add a new command, we
+  don't need to change anything in the Makefile; the binary
+  generated name is the COMMAND directory name. Example, if I
+  have the file `cmd/nice-command/main.go`, when I invoke the
+  `build` rule I will get the `release/nice-command` binary
+  if the compilation is succesful, and no changes are needed
+  to the Makefile if a new `cmd/other-nice-command/main.go`
+  is created. "Write once, use forever".
+- `mk/docker.mk`: Generic rules to build and push container images.
+  It has an interesting set of variables that combined properly
+  allow us a lot of customization and create new rules on them.
+- `mk/db.mk`: Rules to start/stop a postgres container database
+  and clean the data volume. It adds some other rules to quickly
+  invoke the migration and seed the database with random values.
+- `mk/plantuml.mk`: Invoke plantuml to generate diagrams in SVG
+  format from the `docs/*.puml` files. It requires that plantuml
+  is installed in your environment. Add a generic rule for it, so
+  that no changes are needed when new `.puml` file is added to the
+  `doc/` directory.
+
+@Miscelanea
+
+- `mk/printvars.mk`: It helps to fix wrong behaviors so it print out
+  the variable name and the value not evaluated, so we can realize
+  what final values could be applied and how should the values be
+  expanded.
+- `mk/alias.mk`: It provides compatibility with the initial set of
+  rules in the Makefile, so that it would avoid impact during a
+  transition period to the new set of rules.
```

Comment:
our project is so new, i think we should just drop the aliases alltogether, but i'll bring this up in our next standup :)

Reply:
thanks! it does not make many sense the `mk/alias.mk` then, but keeping it until the next standup.